src/System.Net.HttpListener/src/System/Net/HttpHeaderStrings.Unix.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net
+{
+    internal class HttpHeaderStrings
+    {
+        internal const string Authorization = "Authorization";
+        internal const string ContentType = "Content-Type";
+        internal const string Charset = "charset=";
+        internal const string Server = "Server";
+        internal const string NetCoreServerName = ".NETCore-HTTPAPI/1.0";
```

Comment:
We should leave the old value then since the main purpose is app-compat.
If we don't care about app-compat here, the string contains HTTPAPI/1.0 which indicates HTTP.SYS from Windows 2003 (https://blogs.msdn.microsoft.com/vijaysk/2010/09/01/fingerprinting-iis/) and so should be changed to something else (maybe `NetCore` only).